recipes/prefsync/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "1.2" %}
+{% set sha256 = "66f0d3870b442e715de23b632e83f27f5971885b40bb3f6b211437138c554496" %}
+
+package:
+  name: prefsync
+  version: {{ version }}
+
+source:
+  fn: prefsync-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/s/prefsync/prefsync-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Also to clarify my terse phone comment, this should be as follows to build on OS X only.

```
  skip: true  # [not osx]
```